workers/ohsome_quality_analyst/indicators/ghs_pop_comparison_buildings/indicator.py

```diff
@@ -71,7 +68,7 @@ def calculate(self) -> bool:
         )
 
         if self.pop_count_per_sqkm == 0:
-            return False
+            return None
```

Comment:
I think if you don't intend to ever return a value, which you imply by the type hint, you would only use `return` here. See also this comparison: https://stackoverflow.com/a/15300671/3893856